### ISSUE TITLE
inspector_pause test flaky

### ISSUE THREAD

**Issue by ry** [denoland/deno#4589]
https://github.com/denoland/deno/runs/555522900
```
---- inspector_pause stdout ----
target_dir D:\a\deno\deno\target\release
ws_url ws://127.0.0.1:9230/ws/32db3e15-0d08-4ee4-af90-b3f9ca5a7273
##[error]thread 'inspector_pause' panicked at 'called `Result::unwrap()` on an `Err` value: Io(Os { code: 10054, kind: ConnectionReset, message: "An existing connection was forcibly closed by the remote host." })', cli\tests\integration_tests.rs:2076:17
```

**Comment by avillega** [comment]
I would like to work on this one!

**Comment by avillega** [comment]
Looks like it got fixed by c0cb198114ccc2dc4fa2764d307ad985c456882a I ran the tests several times and there was no failures. I created a PR to re-enable the tests
